src/shapes/Object/ObjectGeometry.ts

```diff
@@ -343,10 +342,8 @@ export class ObjectGeometry<EventSpec extends ObjectEvents = ObjectEvents>
    * @return {Boolean} true if point is inside the object
    */
   containsPoint(point: Point, absolute = false, calculate = false): boolean {
-    return cornerPointContainsPoint(
-      point,
-      this._getCoords(absolute, calculate)
-    );
+    const { tl, tr, br, bl } = this._getCoords(absolute, calculate);
```

Comment:
@ShaMan123 please not that your test implementation was based on this.getCoords, while for compatibility reason this is still using _getCoords.

There is a lot to say about moving this to getCoords, and i don't want to do it now

Reply:
I fixed in in #9372